.gitignore

```
@@ -16,3 +16,9 @@ build/*
 tmp/*
 package/DEBIAN/control
 *.swp
+.idea/.gitignore
+.idea/ckanext-dcat.iml
+.idea/misc.xml
+.idea/modules.xml
+.idea/vcs.xml
+.idea/inspectionProfiles/profiles_settings.xml
```

gitignore on folder level gives less noise

```suggestion
.idea
```